docs/general-usage.md

```diff
@@ -0,0 +1,120 @@
+# General Usage
+
+## Contents
+[Configure and Build the service](#configure-and-build-the-service)  
+[Run device-usb-service](#run-device-usb-camera)  
+[Next Steps](#next-steps)
+
+NOTE: An additional package may be required to build the device-use-camera service
+```
+ sudo dpkg -i linux-libc-dev_5.10.0-14.15_amd64.deb
```

Comment:
We should check to make sure this is still needed.

Reply:
i'm going to leave it in for now